File: clients/kotlin/src/main/kotlin/komus24/MyStrategy.kt
class MyStrategy(constants: komus24.model.Constants) {
    fun getOrder(playerView: komus24.model.PlayerView, debugInterface: DebugInterface?): komus24.model.Order {
        return komus24.model.Order(emptyArray())
    }
    fun debugUpdate(displayedTick: Int, debugInterface: DebugInterface?) {}
    fun finish() {}
}

File: clients/scala/src/main/scala/komus24/MyStrategy.scala
class MyStrategy(constants: komus24.model.Constants) {
  def getOrder(playerView: komus24.model.PlayerView, debugInterface: Option[DebugInterface]): komus24.model.Order =
    komus24.model.Order(Seq.empty)
  def debugUpdate(displayedTick: Int, debugInterface: DebugInterface): scala.Unit = ()
  def finish(): scala.Unit = ()
}